### PR TITLE
refactor(simple-color-picker): move from experimental to components - FE-2722

### PR DESCRIPTION
**Pull request by dawidzarzycki**
BREAKING CHANGE: SimpleColorPicker has moved to src/components/simple-color-picker

### Proposed behaviour
new path for SimpleColorPicker

### Current behaviour
SimpleColorPicker is in Experimental 

### Checklist

- [x] Commits follow our style guide
- [ ]  Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

**Review by sianford — DISMISSED** [review]
Reviewed Cypress changes and tested changed feature files - all look correct ✅

**Review by edleeks87 — DISMISSED** [review]

**Review by DipperTheDan — APPROVED** [review]

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature [hunk 1]
```diff
@@ -104,28 +99,28 @@ Feature: Accessibility tests - Design System folder
...
-      | Textarea          |
-      | Textbox           |
-      | Text Editor       |
-      | Tile Select       |
+      | component             |
+      | Advanced Color Picker |
+      | Badge                 |
+      | Batch Selection       |
+      | Content               |
+      | Draggable             |
```
Comment (edleeks87) [inline cypress/features/regression/accessibility/accessibilityDesignSystem.feature:107]: This is missing `Date Input` and `Date Range`

**Review by edleeks87 — CHANGES_REQUESTED** [review]
Couple of components missing from accessibility cypress tests

File: cypress/features/regression/accessibility/accessibilityDesignSystem.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/accessibility/accessibilityDesignSystem.feature:107]: Those tests are / should be in `accessibilityCommon.feature` file

File: cypress/features/regression/designSystem/simpleColorPicker.feature [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Feature: Simple Color Picker component
-  I want to test Simple Color Picker component
+Feature: Design System  Simple Color Picker component
```
Comment (DipperTheDan) [inline cypress/features/regression/designSystem/simpleColorPicker.feature:1]: Looks like there is a double space here.

File: cypress/features/regression/designSystem/simpleColorPicker.feature [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+  I want to test Design System  Simple Color Picker component
```
Comment (DipperTheDan) [inline cypress/features/regression/designSystem/simpleColorPicker.feature:2]: and here too.

**Review by DipperTheDan — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by sianford — DISMISSED** [review]

**Comment by sianford** [comment]
Tested in Chrome. No regressions. Testing passed ✅

**Review by sianford — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Pull request merged**